stories/demos/components/Editors/SelectEditor.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useRef } from 'react';
+import Select, { OptionTypeBase, OptionsType } from 'react-select';
+
+interface SelectEditorProps {
+  value: string;
+  onChange: (value: string) => void;
+  options: OptionsType<OptionTypeBase>;
+  rowHeight: number;
+  menuPortalTarget: Element;
+}
+
+export function SelectEditor<R>({ value, onChange, options, rowHeight, menuPortalTarget }: SelectEditorProps) {
+  const selectRef = useRef<Select<OptionTypeBase>>(null);
+  return (
+    <Select
+      autoFocus
+      defaultMenuIsOpen
+      ref={selectRef}
+      value={options.find(o => o.value === value)}
+      onChange={o => onChange(o.value)}
```

Comment:
When the value is selected, this select editor is not closed.

I am using my mac, haven't tested with Windows yet.

Reply:
That is the default behavior but we can change the editor to call `onCommit` when the value is changed I believe. Or may be we can change `onChange(newValue, commit flag)`. Wdyt?

Reply:
Maybe just keep the `dropdown` open after `onChange`?